### ISSUE TITLE
The files sort may cause that the README.md file is after the child directory

### ISSUE THREAD

**Issue by zpfang** [julianxhokaxhiu/gitbook-plugin-summary#8]
```
const sameDir = path.dirname(a) == path.dirname(b)
            if (sameDir && isReadme(a))
              return -1
            if (sameDir && isReadme(b))
              return 1
            return a < b ? -1 : a > b ? 1 : 0
```

In this code 
if the directory like this:
```
├── README.md
│   ├── ADIC
               └── TEST.md
```
the files  is like this:
README.md 
ADIC/TEST.md

the summery may like this:
(TEST)[ADIC/TEST.md]
(READ)[README.md ]

but it's not my expect

I think that the 'sameDir ' function should judge
```
 path.dirname(b).index(path.dirname(a)) != -1
```

**Comment by julianxhokaxhiu** [comment]
Hi,

have you got a PR and/or a branch where this works? I would be happy to try it. Maybe with a provided test case.

Thank you in advance,
Julian

**Comment by zpfang** [comment]
Hi Julian,
beacase I have not have the guide of the gitbook plugin developig, so I can't try my think in the gitbook enviroment,  where are you learning about the gitbook pulugins? can you share it to me ? 

But lately, I found that the summary plugin doesn't work in the first executing the gitbook serve command, the secode one it works, the first one uses the summary file generated by the laste time.

Thank you for your reply,
zpfang

**Comment by julianxhokaxhiu** [comment]
Solved with #9

**Comment by julianxhokaxhiu** [comment]
Released as `0.2.1`
